### ISSUE TITLE
Django 1.8 - django.db.utils.ProgrammingError: relation "auth_permission" does not exist

### ISSUE THREAD

**Issue by ebugfix** [django-guardian/django-guardian#304]
**Django 1.8** version 

django-guardian rise this error when I try to use with my custom user model.

This happen to perform manage.py makemigrations.

*settings.py*
(AUTH_USER_MODEL = Users.user*

Traceback (most recent call last):
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
psycopg2.ProgrammingError: relation "auth_permission" does not exist

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/src/manage.py", line 11, in <module>
    execute_from_command_line(sys.argv)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/core/management/base.py", line 390, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/core/management/base.py", line 441, in execute
    output = self.handle(*args, **options)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/core/management/commands/migrate.py", line 179, in handle
    created_models = self.sync_apps(connection, executor.loader.unmigrated_apps)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/core/management/commands/migrate.py", line 317, in sync_apps
    cursor.execute(statement)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/db/utils.py", line 97, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/utils/six.py", line 658, in reraise
    raise value.with_traceback(tb)
  File "/home/cfpp/workspace/cerebromix/src/dev-backend/lib/python3.4/site-packages/django/db/backends/utils.py", line 62, in execute
    return self.cursor.execute(sql)
django.db.utils.ProgrammingError: relation "auth_permission" does not exist

**Comment by phanviet** [comment]
It's not compatable with django 1.8. Please downgrade to django 1.7

**Comment by rowanseymour** [comment]
Am getting same error with latest master. Is there any workaround for now so we don't have to downgrade to django 1.7?

**Comment by rowanseymour** [comment]
Ok seems I wasn't using latest master. If you are using pip you can change your requirement to...

```
-e git+https://github.com/lukaszb/django-guardian#egg=django-guardian
```

Run the new guardian migrations like...

```
python manage.py migrate guardian --fake-initial
```

And this error goes away

**Comment by alexandervlpl** [comment]
Does anyone know how this issue was fixed in guardian? My application has the exact same problem, I'm looking at the code and the commits and trying to figure out how to get rid of it.

**Comment by harveydf** [comment]
@rowanseymour It works on 1.8, thank you

**Comment by alexandervlpl** [comment]
Here's a pip requirement for the specific commit that seems to have introduced full 1.8 support:
`-e git+https://github.com/lukaszb/django-guardian@745772f878743219e97790c2e944a492d8d85915#egg=django-guardian`

Works for me while I wait for the next release.

**Comment by brianmay** [comment]
I believe this was fixed in 745772f878743219e97790c2e944a492d8d85915, which was part of the 1.3 release. Closing.
